lib/ProxyBus.ts

```diff
@@ -21,7 +20,7 @@ export class ProxyBus implements EventBus {
     }
 
     getVersion(): string {
-        return packageJson.version
+        return globalThis.__pkg_version;
```

Comment:
I've learned something new

Reply:
@artonge it is not a real global variable.
It will be replaced in bundle file.

![image](https://user-images.githubusercontent.com/1561347/165964164-acfd14f9-bab8-4705-b556-f8a6da51bf0a.png)

Reply:
It will run in the browser environment @skjnldsv, not in node like webpack config script.

I am only able to change it from rollup config, but there are some issues to use `process.env` and ESM syntax (like we are using in `rollup.config.js`)

Unfortunately, ESM and CJS are very incompatible and generate a lot of problems when we mix those things...

Reply:
> It will run in the browser environment @skjnldsv, not in node like webpack config script.

Yes, if you define as a global variable in rollup, it's good :)
https://www.npmjs.com/package/@rollup/plugin-replace